lib/afterpay/refund.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module Afterpay
+  # The Refund object
+  class Refund
+    # refund_id [string]: unique ID returned by afterpay after successful refund
```

Comment:
move these lines after Line 15, also add another line like this

# == Response [Refund] Object
# refund_id [string]: unique ID returned by afterpay after successful refund
# refunded_at [timestamp]: The timestamp at which the refund was done
# error [hash]: error information if refund fails